### ISSUE TITLE
Beeg broken again

### ISSUE THREAD

**Issue by Kodionpi** [dobbelina/repository.dobbelina#602]
### Bug reports:
Beeg is not working at all

### OS environment
Kodi 18.9 Libreelec 9.2.6, Raspberry Pi 2b

[kodi1.log](https://github.com/dobbelina/repository.dobbelina/files/7188602/kodi1.log)

**Comment by Gujal00** [comment]
You already have #589 open for this same issue, why open multiple issues for the same?

**Comment by Kodionpi** [comment]
That is correct, but then an update came and the error was fixed. However, recently it does not work again. 

"with no debug log" What do you mean? There is a log file attached.

**Comment by Gujal00** [comment]
yeah need full uncut debug log from kodi start to actual failure in DEBUG mode, not a snippet of a log

**Comment by dobbelina** [comment]
With some luck it works now after the update.

**Comment by Kodionpi** [comment]
@Gujal00 I see, I had thought that it was sufficient to post only the essential part of the log (i.e. where the error occurs). I will remember that next time. Thanks.

I thought that the previous error message of mine already said "fixed",- after the last update. I had reported the error 18 days ago and 14 days ago the update came. Through this update it worked again for a few days. Then suddenly no longer and (believing that the previous error message was marked "fixed") I posted renewed. Was obviously a misunderstanding on my part. Sorry about that.

@dobbelina Thanks for working a new update.

**Comment by Kodionpi** [comment]
@dobbelina Thanks a lot for your work!!! I have now installed the new update and now everything runs perfectly. One tiny little thing I noticed. I have set FHD as the preferred maximum quality in the settings. This logically plays all 4k/2160p as FHD/1080p. This works very well. However, there are very few videos where 2024p is specified. For these, the 1080p variant is not played as specified in the settings. -but it remains at 2024p, which some devices can not handle.
